hamlpy/hamlpy.py

```diff
@@ -83,38 +84,17 @@ def process(self, haml):
         """
         Converts the given string of Haml to a regular Django HTML
         """
-        return self.process_lines(haml.split('\n'))
+        stream = Stream(haml)
 
-    def process_lines(self, haml_lines):
-        """
-        Converts the given list of lines of Haml to a regular Django HTML
-        """
         root = Node(self.options)
-        line_iter = iter(haml_lines)
-
-        haml_node = None
-        for line_number, line in enumerate(line_iter):
-            node_lines = line
-
-            if not root.parent_of(HamlNode(line, self)).inside_filter_node():
-                if line.count('{') - line.count('}') == 1:
-                    start_multiline = line_number  # for exception handling
```

Comment:
Tracking line numbers would be useful but this line doesn't actually do anything